src/docfx/build/Build.cs

```diff
@@ -51,7 +51,7 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Repo
                 }
             }
 
-            // await saveGitHubUserCache;
+            await saveGitHubUserCache;
```

Comment:
>saveGitHubUserCache [](start = 18, length = 19)

why not just await githubUserCache.SaveChanges();

Reply:
So the code in between can be executed when saveGitHubUserCache in parallel

---
In reply to: [223551772](https://github.com/dotnet/docfx/pull/3467#discussion_r223551772) [](ancestors = 223551772)